### PR TITLE
Atom 1.20+: Add Vega transform

### LINKED ISSUE #560: Altair visualization library doesn't work
Hello!

I've been working with your package lately and it's really great. Unfortunately, it fails to display the output of the [Altair](https://altair-viz.github.io/) library. I try running this code from their homepage:

```python
from altair import Chart, load_dataset

# load built-in dataset as a pandas DataFrame
cars = load_dataset('cars')

Chart(cars).mark_circle().encode(
    x='Horsepower',
    y='Miles_per_Gallon',
    color='Origin',
)
```

## Notebook

screen shot 2016-12-27 at 01 35 51

## Hydrogen

screen shot 2016-12-27 at 01 34 21

## Notebook output contents

```html
<div class="output_subarea output_html rendered_html"><div class="vega-embed" id="c21b9d22-57b7-4cc8-9edf-bac5abf1c864">
  <div class="vega" style="position: relative;">
    <canvas width="624" height="405" class="marks">
    </canvas>
  </div>
  <div class="vega-actions">
    <a href="#" target="_blank" download="vega.png">Export as PNG</a>
    <a href="#">View Source</a>
    <a href="#">Open in Vega Editor</a>
  </div>
</div>

<style>
.vega-embed svg, .vega-embed canvas {
  border: 1px dotted gray;
}

.vega-embed .vega-actions a {
  margin-right: 6px;
}
</style>
</div>
```

## Hydrogen output contents

```html
<div class="bubble-output-container">
  <div class="bubble-result-container">
    <div class="vega-embed" id="c8ff3624-14a1-4ad8-b422-bbaa30bdba13">
    </div>

    <style>
      .vega-embed svg, .vega-embed canvas {
        border: 1px dotted gray;
      }

      .vega-embed .vega-actions a {
        margin-right: 6px;
      }
    </style>
  </div>
  <div class="bubble-error-container plain-error">
  </div>
  <div class="bubble-status-container hidden" style="height: 21px">
    ...
  </div>
</div>
```

I am under the impression that the package that is used to embed Altair's graphics is https://github.com/vega/ipyvega, but I am not familiar with Altair's source enough to tell for sure.

So, does support for this library have to be added to Hydrogen by hand or is something broken here?

### PR DESCRIPTION
**Pull request by lgeiger**
⚠️ **DON'T MERGE BEFORE ATOM 1.20 IS STABLE** ⚠️ 

The CSP of Atom 1.20 now allows `unsafe-eval`. Therefore we're able to remove the hack for plotly and support Vega 🎉 

This PR also switches to the new `observeActiveTextEditor`.

Closes #560

### CODE REVIEW

**Comment by rgbkrk** [comment]
I think that Altair is going to release soon with the custom mimetypes, which simplifies our examples here. I'm stoked!

**Comment by lgeiger** [comment]
> I think that Altair is going to release soon with the custom mimetypes, which simplifies our examples here. I'm stoked!

Yeah, added a [example for Altair 1.3+](https://github.com/nteract/hydrogen/pull/939/files#diff-8d62465ddfc3a8dd416ea79898b3989cR72)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ We use [Jasmine](https://jasmine.github.io/2.5/introduction) for writing specs.
 To build the website locally run:
 ```bash
 npm install gitbook-cli -g
+gitbook install
```
Comment (BenRussert) [inline CONTRIBUTING.md]: We may want to merge this line before v1.20 :grin:

**Comment by lgeiger** [comment]
Atom 1.20 is now stable so this is ready to merge 😄

**Comment by rgbkrk** [comment]
Nice! I've just updated mine. 😄

File: docs/Usage/Examples.md [hunk 2]
```diff
@@ -53,6 +53,33 @@ t = np.linspace(0, 20, 500)
 
 plt.plot(t, np.sin(t))
 plt.show()
+{%- language name="Python using altair < v1.3", type="py" -%}
+from IPython.display import display
+from altair import Chart, load_dataset
+def vegify(spec):
+    display({
+        'application/vnd.vegalite.v1+json': spec.to_dict()
+    }, raw=True)
```
Comment (rgbkrk) [inline docs/Usage/Examples.md]: I can't wait for ipyvega to output the proper mimebundle for this.

**Pull request merged**